src/material/form-field/form-field.ts

```diff
@@ -582,4 +582,20 @@ export class MatFormField extends _MatFormFieldMixinBase
   private _getStartEnd(rect: ClientRect): number {
     return this._previousDirection === 'rtl' ? rect.right : rect.left;
   }
+
+  /** Checks whether the form field is attached to the DOM. */
+  private _isAttachedToDOM(): boolean {
```

Comment:
Do we currently use a similar pattern elsewhere in the library? If so might make sense to put as a utility in cdk or material/core

Reply:
I did a quick a search and it doesn't look like we do.